src/rules/no-disabled-tests.ts

```diff
@@ -1,5 +1,5 @@
-import { createRule } from './tsUtils';
-import { getNodeName, scopeHasLocalReference } from './tsUtils';
+import { createRule } from './utils';
+import { getNodeName, scopeHasLocalReference } from './utils';
```

Comment:
merge these imports into a single statement (I realize it's old)

Reply:
No problem - IntelliJ should have done that, but I guess I missed that one when renaming `tsUtils` :joy:

Reply:
No problem - IntelliJ should have done that, but I guess I missed that one when renaming `tsUtils` :joy: